packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/nodejs/runwidgetapp.md

```diff
@@ -10,12 +10,20 @@ The sample app is located here: `okta-auth-js/samples/generated/express-embedded
 1. Open a browser window and navigate to the app's home page at `http://localhost:8080`. After the app loads, click the **Sign In** button located on the home screen.
 1. On the sign-in page, enter the username (email) and password you used in [Create your Okta account](/docs/guides/oie-embedded-common-org-setup/nodejs/main/#create-your-okta-account).
 
+   <div class="half">
+
    ![Displays the Sign-In Widget](/img/oie-embedded-sdk/oie-embedded-widget-use-case-sign-in-screen.png)
 
+   </div>
+
 1. If you've successfully signed in, the app redirects you to the user profile page that displays basic user profile and security token information.
 
+   <div class="three-quarter">
```

Comment:
This needs a border else half of it disappears in dark mode.
```suggestion
   <div class="three-quarter border">
```

Reply:
Hrm, I've tried removing the border radius, and think it looks much better.